src/ploneintranet/workspace/profiles/default/workflows/ploneintranet_workspacecontainer_workflow/definition.xml

```diff
@@ -0,0 +1,100 @@
+<?xml version="1.0"?>
+<dc-workflow workflow_id="ploneintranet_workspacecontainer_workflow"
+             title="Ploneintranet Workspace Container Workflow"
+             description="- Workspace Container Workflow"
+             state_variable="review_state"
+             initial_state="open"
+             manager_bypass="False">
+
+  <permission>Add portal content</permission>
+  <permission>Access contents information</permission>
+  <permission>List folder contents</permission>
+  <permission>Modify portal content</permission>
+  <permission>View</permission>
+
+  <state state_id="open"
+         title="Open">
+    <description>Viewing and content created open for all authenticated members.</description>
+    <exit-transition transition_id="restrict"/>
+    <permission-map name="Add portal content"
+                    acquired="False">
+      <permission-role>Authenticated</permission-role>
+    </permission-map>
+    <permission-map name="Access contents information"
+                    acquired="False">
+      <permission-role>Authenticated</permission-role>
+    </permission-map>
+    <permission-map name="List folder contents"
+                    acquired="False">
+      <permission-role>Authenticated</permission-role>
+    </permission-map>
+    <permission-map name="Modify portal content"
+                    acquired="False">
+      <permission-role>Authenticated</permission-role>
```

Comment:
This is a problem. We should not allow any authenticated user to modify the workspacecontainer. This should be removed and just acquire the sitewide ModifyPortalContent settings.

Additionally, only Manager, not even Site Manager, should be allowed to create or delete workspacecontainers. This requires a special permission. I've just fixed a comparable pattern in another site. The workspacecontainer is more like a site utility than normal content. The code relies on it being present. Deleting it is catastrophic. Adding another one is a big decision. Most site managers I'm working with cannot be trusted to not f*ck this up. They have, and they will, if you don't protect them against themselves.